src/app/cli/src/graphql.ml

```diff
@@ -1159,7 +1188,13 @@ module Mutations = struct
           User_command_payload.Body.Payment
             {receiver; amount= Amount.of_uint64 amount}
         in
-        build_user_command coda sender_account sender_kp memo body fee )
+        let nonce =
+          Option.value_map nonce_opt
+            ~f:(fun nonce32 ->
+              Account.Nonce.of_int @@ Unsigned.UInt32.to_int nonce32 )
```

Comment:
This conversion is real dubious. On 32 bit platforms there's a possibility of overflow. From looking at the source it looks like these are actually the same type, and the equalities are exposed, so maybe it's a matter of the right `:>`.

Reply:
You couldn't use subtype. Never the less, I resolved this issue.